docs/index.html

```diff
@@ -900,7 +900,7 @@ <h4>CSS Custom Properties (CSS Variables)</h4>
 </code></pre>
 <p>Even though custom properties are not supported by some aging browsers, Animate.css provides a proper fallback, widening its support for any browser that supports CSS animations.</p>
 
-        <p class="edit-github"><a href="https://github.com/daneden/animate.css/blob/master/docsSource/sections/01-usage.md" title="Edit this on Github">Edit this on Github</a></p>
+        <p class="edit-github"><a href="https://github.com/daneden/animate.css/blob/master/docsSource/sections/01-usage.md" title="Edit this on GitHub">Edit this on GitHub</a></p>
```

Comment:
I just realised that the docs are still pointing to `daneden/animate.css`—May as well fix that in this PR?

Reply:
It's amazing the number of small things that pass by when writing a doc page. 🙃

I just updated it.